graph/src/components/ethereum/adapter.rs

```diff
@@ -1,15 +1,17 @@
 use ethabi::{Bytes, Error as ABIError, Function, ParamType, Token};
-use failure::{Error, SyncFailure};
+use failure::SyncFailure;
 use futures::Future;
-use slog::Logger;
+use petgraph::graphmap::GraphMap;
 use std::collections::{HashMap, HashSet};
-use std::iter::FromIterator;
+use std::fmt;
 use tiny_keccak::keccak256;
 use web3::types::*;
 
 use super::types::*;
 use crate::prelude::*;
 
+pub type EventSig = H256;
```

Comment:
Any reason to shorten this name to `EventSig`? How about `EventSignature`? :wink: